2019-09-29 15:01:42.442 [ERR] [session.go:253] recvLoop error: read tcp 127.0.0.1:17000->127.0.0.1:54800: wsarecv: An existing connection was forcibly closed by the remote host., remoteAddr: 127.0.0.1:54800, sessionid: 1, localAddr: 127.0.0.1:17000
2019-09-29 15:01:42.443 [ERR] [session.go:265] read and handle message error: read tcp 127.0.0.1:17000->127.0.0.1:54800: wsarecv: An existing connection was forcibly closed by the remote host., remoteAddr: 127.0.0.1:54800,  localAddr: 127.0.0.1:17000, sessionid: 1
2019-09-29 15:01:42.458 [ERR] [Entity.go:601] AsyncCall service not exist: 39,  sType: 3 ,methodName: SyncEntitySrvInfo
2019-09-29 15:10:10.097 [ERR] [mongodboperate.go:89] session nil, not connect or auth
2019-09-29 15:10:10.1 [ERR] [Entity.go:601] AsyncCall service not exist: 39,  sType: 3 ,methodName: SyncEntitySrvInfo
2019-09-29 15:14:50.419 [ERR] [mongodboperate.go:89] session nil, not connect or auth
2019-09-29 15:17:00.663 [ERR] [session.go:253] recvLoop error: read tcp 127.0.0.1:17000->127.0.0.1:55731: wsarecv: An existing connection was forcibly closed by the remote host., remoteAddr: 127.0.0.1:55731, sessionid: 1, localAddr: 127.0.0.1:17000
2019-09-29 15:17:00.664 [ERR] [session.go:265] read and handle message error: read tcp 127.0.0.1:17000->127.0.0.1:55731: wsarecv: An existing connection was forcibly closed by the remote host., remoteAddr: 127.0.0.1:55731,  localAddr: 127.0.0.1:17000, sessionid: 1
2019-09-29 15:21:06.792 [ERR] [mongodboperate.go:89] session nil, not connect or auth
2019-09-29 15:21:23.306 [ERR] [session.go:253] recvLoop error: read tcp 127.0.0.1:17000->127.0.0.1:55931: wsarecv: An existing connection was forcibly closed by the remote host., remoteAddr: 127.0.0.1:55931, sessionid: 1, localAddr: 127.0.0.1:17000
2019-09-29 15:21:23.307 [ERR] [session.go:265] read and handle message error: read tcp 127.0.0.1:17000->127.0.0.1:55931: wsarecv: An existing connection was forcibly closed by the remote host., remoteAddr: 127.0.0.1:55931,  localAddr: 127.0.0.1:17000, sessionid: 1
2019-09-29 15:22:07.565 [ERR] [mongodboperate.go:89] session nil, not connect or auth
2019-09-29 15:22:07.619 [ERR] [Entity.go:567] AsyncCall Can't call self, methodName: SyncProps
2019-09-29 15:22:07.619 [ERR] [Entity.go:573] AsyncCall GetEntitySrvID error: Entity srvType [6] not existed, methodName: SyncProps
2019-09-29 15:22:12.626 [ERR] [mongodboperate.go:126] session nil, not connect or auth
2019-09-29 15:27:03.876 [ERR] [session.go:253] recvLoop error: read tcp 127.0.0.1:17000->127.0.0.1:56011: wsarecv: An existing connection was forcibly closed by the remote host., remoteAddr: 127.0.0.1:56011, sessionid: 1, localAddr: 127.0.0.1:17000
2019-09-29 15:27:03.878 [ERR] [session.go:265] read and handle message error: read tcp 127.0.0.1:17000->127.0.0.1:56011: wsarecv: An existing connection was forcibly closed by the remote host., remoteAddr: 127.0.0.1:56011,  localAddr: 127.0.0.1:17000, sessionid: 1
2019-09-29 15:27:29.799 [ERR] [mongodboperate.go:89] session nil, not connect or auth
2019-09-29 15:27:29.852 [ERR] [Entity.go:567] AsyncCall Can't call self, methodName: SyncProps
2019-09-29 15:27:29.852 [ERR] [Entity.go:573] AsyncCall GetEntitySrvID error: Entity srvType [6] not existed, methodName: SyncProps
2019-09-29 15:27:34.86 [ERR] [mongodboperate.go:126] session nil, not connect or auth
